aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,73 @@ import { Example } from '../../layouts';
+- Reveal fields progressively so the user is not overwhelmed 
+- Write in sentence case (capitalize the first letter and all following letters should be lowercase)  
+- Fill in default values for as many fields as possible 
+- Use clear, concise title labels for each field 
+- Write helpful placeholder text inside the fields 
+- If the user would likely have doubts, include helpful text below the field, such as [password requirements](#signup) or [additional resources](#customize) on a field's context
+- Place the submit, reset and cancel buttons at the bottom left of the form. [See responsive button behavior](https://design-system.hpe.design/components/button?q=button#responsiveness). 
+- Use specific labels on buttons that tell the user what action to expect 
 
-<ExampleImagePreview
-  label="Required and Optional Fields Example"
-  src="/required&optional-Fields-example.png"
-  href="#required-and-optional-fields"
-  alt="HPE Design System Form example where required and optional fields are used."
-/>
-
-**Forms where 'asterisk' and 'optional' can be skipped**
+**Actions after the user submits information**
+- Provide feedback when the form is submitted successfully (e.g. notification, success page, green check mark, etc.) 
+- [In the case of an error](#requiredoptional), red highlight the field where the error occurred 
+- List below the button all relevant errors and the corrective action the user must take 
+- Allow the user to continue navigating through the application immediately
 
-A login/registration form, which typically shows two to three required fields do not need to be annotated in any way - the form is short and the necessity of completing the fields is relatively obvious.
 
-For login/registration forms, apply `required={{ indicator: false }}` to the FormFields to remove the required indicator.
+## Don'ts
 
-<ExampleImagePreview
-  label="Forms where 'asterisk' and 'optional' can be skipped"
-  src="/form-example-nonrequired.png"
-  href="#sign-in"
-  alt="HPE Design System Form where required and optional can be skipped"
-/>
+**Required fields**
+- Have more than two optional fields 
+- Use asterisks for sign up and sign in forms 
+- Red asterisks stresses the user and actually decrease success in filling out the form 
+- No need to list requirements up front that the user is unlikely to commit (ex. double hyphens, starting with a space, etc.) 
 
-### Using help text
+**Format and fields**
+- Repeat the field label to use as placeholder text 
+- Use generic labels on buttons, such as “Submit” 
 
-In most cases, the FormField label should be clear enough to instruct the user how to complete a field. However, if help text is required, ensure that it is kept brief and precise.
+**Actions after the user submits information**
+- Restrict resource naming. Allow spaces, punctuation and unicode characters (ex. Björk Guðmundsdóttir, 联想, John Q. Public, etc.)
```

Comment:
This bullet doesn't seem related to submission. It feels like it should fall under "Format"

Reply:
This should be kept /fall under users-input guidance

Reply:
Ok, I can see your point. Note: I only put this in that category only because the user often doesn't discover that there was a problem with how they named their resource until after they've clicked on "submit," but I see that there's a case for putting this under the "requirements," category. Thx!